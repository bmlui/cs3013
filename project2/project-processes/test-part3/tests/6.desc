Test || operator, first condition is false 
