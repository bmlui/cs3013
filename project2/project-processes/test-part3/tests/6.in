/usr/bin/false || echo test passed
exit

