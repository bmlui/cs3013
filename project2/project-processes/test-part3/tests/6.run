./wshell < tests/6.in
