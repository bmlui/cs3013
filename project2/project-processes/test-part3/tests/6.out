test-part3$ /usr/bin/false || echo test passed 
test passed 
test-part3$ exit
